doc/src/index.md

```diff
@@ -1,3 +1,36 @@
+```@raw html
+---
+layout: home
+
+hero:
+  text: "The Julia Programming Language"
+  image:
+    light:
+        src: /logo.svg
+        alt: julia
+    dark:
+        src: /logo-dark.svg
+        alt: julia
+  actions:
+    - theme: brand
+      text: Getting Started
+      link: /manual/getting-started
+    - theme: alt
+      text: View on Github
+      link: https://github.com/JuliaLang/julia
+    - theme: brand
+      text: Sponsor
+      link: https://github.com/sponsors/julialang
+---
+```
+
+::: danger
+
+This is a fork from the original repo. Please visit the [official website](https://julialang.org).
+
+:::
```

Comment:
This should not work, since this is not valid Julia Markdown (well, it's valid, but should just render as text). And so if VitePress renders this, that's a bit of a problem.

Reply:
this is vitepress specific https://vitepress.dev/guide/markdown#custom-containers, but the default admonitions also work see https://github.com/LuxDL/DocumenterVitepress.jl/issues/22

Reply:
From my experience with DimensionalData.jl this extra syntax is the best thing about vitepress, this is also how you make tabbed sections with very minimal work.